extensions/wikia/PortabilityDashboard/PortabilityDashboard.setup.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$dir = dirname( __FILE__ ) . '/';
+
+$wgAutoloadClasses[ 'PortabilityDashboardModel' ] = $dir . 'PortabilityDashboardModel.class.php';
+$wgAutoloadClasses[ 'PortabilityDashboardHooks' ] = $dir . 'PortabilityDashboardHooks.class.php';
+$wgAutoloadClasses[ 'SpecialPortabilityDashboardController' ] = $dir . 'SpecialPortabilityDashboardController.php';
+
+$wgSpecialPages[ 'PortabilityDashboard' ] = 'SpecialPortabilityDashboardController';
+$wgSpecialPageGroups[ 'PortabilityDashboard' ] = 'wikia';
```

Comment:
I think that the permissions for this page can safely reflect ones for Insights, like here:
https://github.com/Wikia/config/blob/a990e114914b6ceabcabe6b4d7864cee2760a1fe/PermissionsDefinesAfterWikiFactory.php#L446